Buildings/Experimental/DHC/Examples/Combined/Generation5/ThermalStorages/BoreField.mo

```diff
@@ -0,0 +1,43 @@
+within Buildings.Experimental.DHC.Examples.Combined.Generation5.ThermalStorages;
+model BoreField "Geothermal borefield model"
+  extends Buildings.Fluid.Geothermal.Borefields.TwoUTubes(
+    final energyDynamics=Modelica.Fluid.Types.Dynamics.FixedInitial,
+    final tLoaAgg(displayUnit="h") = 3600,
+    final nSeg=5,
+    TExt0_start=282.55,
+    final z0=10,
+    final dT_dz=0.02,
+    final dynFil=true,
+    borFieDat(
+      final filDat=Buildings.Fluid.Geothermal.Borefields.Data.Filling.Bentonite(
+          kFil=2.0,
+          cFil=3040,
+          dFil=1450),
+      final soiDat=Buildings.Fluid.Geothermal.Borefields.Data.Soil.SandStone(
+          kSoi=2.3,
+          cSoi=1000,
+          dSoi=2600),
+      final conDat=
+          Buildings.Fluid.Geothermal.Borefields.Data.Configuration.Example(
+          borCon=Buildings.Fluid.Geothermal.Borefields.Types.BoreholeConfiguration.DoubleUTubeParallel,
+          dp_nominal=35000,
+          hBor=250,
+          rBor=0.19,
+          nBor=350,
+          cooBor={dBor*{mod(i - 1, 10),floor((i - 1)/10)} for i in 1:350},
+          rTub=0.04,
+          kTub=0.4,
+          eTub=0.0037,
```

Comment:
@mwetter While documenting this model, I realized that the borehole characteristics are a little out of charts.

- `hBor=250` whereas bore depths rather range from 15 to 180 m.
- `rBor=0.19` (so 380 mm diameter) whereas boreholes are typically 100 to 150 mm in diameter.
- `rTub=0.04` (so 80 mm diameter) whereas vertical tubes range from 20 to 40 mm nominal diameter.
- `kTub=0.4` which seems more like the conductivity of PE rather than HDPE (0.48 W/m.K).

Do you remember the rationale behind those values? 
Otherwise I will change them to more typical values and assess the impact on the simulation results.

Reply:
I don't recall the rationale, but the radius vs diameter seems to be a typo. Could you please update all the values to more reasonable settings.

Reply:
I tested with updated values within the typical ranges I mentioned (from 2011 ASHRAE Handbook—HVAC Applications 34.10) but the simulation of `Buildings.Experimental.DHC.Examples.Combined.Generation5.Examples.SeriesVariableFlow` is significantly impacted and would require resizing the borefield.
In addition, the number and depth of the boreholes are specified in

> The reservoir network: A new network topology for district heating and cooling

As I am not 100% sure that those parameter values are not technologically possible, I propose to keep them unchanged and merge as is.